app/Mail/EpistolaCollegii.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Mail;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+
+class EpistolaCollegii extends Mailable
+{
+    use Queueable, SerializesModels;
+
+    /**
+     * Create a new message instance.
+     *
+     * @return void
+     */
+    public function __construct($epistolas)
+    {
+        $this->epistolas = $epistolas;
```

Comment:
Shouldn't `epistolas` and `theme` be declared as a member beforehand?

Reply:
Idk, I will do that. The theme is already a member of the parent class.